README.rst

```diff
@@ -76,6 +84,9 @@ Alternatively, clone and build from source::
     pip install -r requirements.txt
     python setup.py install
 
+When building from source, the closed-source component still needs to be
+installed as above.
```

Comment:
I would just add the command here too in the code line: `pip install dwave-inspectorapp --extra-index=https://pypi.dwavesys.com/simple`

Reply:
I was actually thinking to remove this "build from source" section completely. So I'm hesitant to duplicate the closed-source instructions.